packages/common/array/src/has-many-items.ts

```diff
@@ -1,7 +1,13 @@
 import { Nullable } from '@angular-ru/common/typings';
 
-export function hasManyItems<EntryType>(
-    array?: Nullable<EntryType[]>
-): array is [EntryType, EntryType, ...EntryType[]] {
-    return (array ?? []).length > 1;
+import { ListType } from './types';
+
+export function hasManyItems<EntryType>(list?: Nullable<ListType<EntryType>>): boolean {
+    if (Array.isArray(list)) {
+        return (list ?? []).length > 1;
```

Comment:
as I remember, `Array.isArray` should return `false` if `null` or 'undefined' provided, so there is no need to check it again using `??`